env.example

```diff
@@ -0,0 +1,109 @@
+#
+#PORT=8015
+
+#
+#WS_SERVER_PATH='/jitsi-component-selector/ws'
+
+#
+#LOG_LEVEL='info'
+
+#
+#REDIS_HOST='127.0.0.1'
+
+#
+#REDIS_PORT=6379
+
+#
+#REDIS_PASSWORD=''
+
+#
+#REDIS_TLS=false
+
+#
+#REDIS_DB=0
+
+#
+#REDIS_SCAN_COUNT=100
+
+#
+#PROTECTED_API=true
+
+#
+#COMPONENT_TTL_SEC=3600
+
+#
+#CANDIDATE_TTL_SEC=120
+
+#
+#SESSION_TTL_SEC=86400
+
+#
+#ASAP_PUB_KEY_TTL=3600
+
+# (required)
+# e.g. [{"kid": "kidPattern", baseUrl": "https://asap.org"}]
+SYSTEM_ASAP_BASE_URL_MAPPINGS=
+
+# The audience value to be used in asap jwt tokens (required)
+SYSTEM_ASAP_JWT_AUD=
+
+# (required)
+SYSTEM_ASAP_JWT_ACCEPTED_HOOK_ISS=
+
+# (required)
+# e.g. [{"kid":"kidPattern","baseUrl":"https://asap.org"]
+JITSI_ASAP_BASE_URL_MAPPINGS=
+
+# The value to be used in audience field of jitsi jwt token (required)
+JITSI_ASAP_JWT_AUD=
+
+# (required)
+JITSI_ASAP_JWT_ACCEPTED_HOOK_ISS=
+
+#
+#KEY_PREFIX_PATTERN='^(.*)/(.*)$'
+
+#
+#SIP_ADDRESS_PATTERN='(^sips?:)?(.*)(@.*)'
+
+#
+#SIP_JIBRI_INBOUND_EMAIL='inbound-sip-jibri@jitsi.net'
```

Comment:
Do we want to leave this as the default value? Or we can leave it empty and skip functionality if missing?

Reply:
We can leave it empty